src/runtime-rs/crates/resource/src/rootfs/mod.rs

```diff
@@ -49,15 +55,25 @@ impl RootFsResource {
     pub async fn handler_rootfs(
         &self,
         share_fs: &Option<Arc<dyn ShareFs>>,
+        hypervisor: &dyn Hypervisor,
+        sid: &str,
         cid: &str,
         bundle_path: &str,
         rootfs_mounts: &[Mount],
     ) -> Result<Arc<dyn Rootfs>> {
         match rootfs_mounts {
+            mounts_vec if rootfs_mounts[0].fs_type == NYDUS_ROOTFS_TYPE => {
```

Comment:
I think this should move the branch `mounts_vec if is_single_layer_rootfs(mounts_vec)`.

Reply:
fixed